src/Product.php

```diff
@@ -9,6 +9,13 @@ class Product {
 
   const CATEGORY_EXCLUDED = ['Ausstellungsstücke', 'Abverkauf'];
 
+  /**
+   * Stores the stock of the products across all variations available.
```

Comment:
Please add a PHPDoc description that explains for what reason this is done.

Reply:
Updated in the last commit.